### PR TITLE
Rework the job arguments

### PR DESCRIPTION
**Pull request by tfiedor**
This PR:

1. Removes `args` from existence, and merge their functionality into `cmd`

### CODE REVIEW

File: tests/test_fuzzing.py [hunk 1]
```diff
@@ -348,7 +347,6 @@ def test_fuzzing_incorrect(pcs_with_root):
         cli.fuzz_cmd,
         [
             "--cmd", "ls",
-            "--args", "-al",
             "--output-dir", ".",
         ],
     )  # fmt: skip
```
Comment (JiriPavela) [inline tests/test_fuzzing.py:352]: Shouldn't the `--args` value be appended to the `--cmd`, or doesn't it matter for those tests?

**Review by JiriPavela — APPROVED** [review]
Ok, let's hope we won't later discover we actually need the `--args`.

**Comment by tfiedor** [comment]
> Ok, let's hope we won't later discover we actually need the `--args`.

No we will not, and `cmd` is fine substitution. I think in future we might rather go for multiple (named) job matrixes instead (I wanted just some quick refactoring, so I omited the refactoring of job matrix).

File: tests/test_fuzzing.py (hunk 1 above)
Comment (tfiedor) [reply tests/test_fuzzing.py:352]: Good point, but it should not matter.

**Pull request merged**